Source/src/Batching/BatchManager.cpp

```diff
@@ -54,7 +61,7 @@ void Hachiko::BatchManager::CollectMesh(const GameObject* game_object)
                 // Create new batch if there are no matching ones
                 GeometryBatch* new_batch = new GeometryBatch(resource->layout);
                 new_batch->AddMesh(mesh_renderer);
-                geometry_batches.push_back(new_batch);
+                (*geometry_batches).push_back(new_batch);
```

Comment:
geometry_batches)->push_back(new_batch); // isn't using the pointer expression more clear?

Reply:
@Bernatmago added the required changes, I'm resolving the convo.